src/repl.jl

```diff
@@ -47,7 +47,12 @@ function rewrite_with_ANSI(s, cursormove::Bool = false)
 
         l = textwidth(get_prompt(s))
         if !isa(s, LineEdit.SearchState)
-            LineEdit.write_prompt(termbuf, mode)
+            # xref: https://github.com/JuliaLang/julia/pull/36689
+            @static if VERSION ≥ v"1.6.0-DEV.531"
+                LineEdit.write_prompt(termbuf, mode, LineEdit.hascolor(terminal(s)))
```

Comment:
Not sure `LineEdit.hascolor` stuff is necessary; maybe everyone who uses this package uses julia with `--color=yes`.